src/Directive/FigureDirective.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Docs Builder package.
+ * (c) Ryan Weaver <ryan@symfonycasts.com>
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace SymfonyDocsBuilder\Directive;
+
+use Doctrine\RST\Directives\Directive;
+use Doctrine\RST\Nodes\Node;
+use Doctrine\RST\Parser;
+
+class FigureDirective extends Directive
```

Comment:
Change this to a `SubDirective` - THEN your `processSub()` should be called. A SubDirective is one where you want the "inside" of the directive to be parsed like the rest of RST - https://github.com/doctrine/rst-parser/blob/9107ba8b1dc8dc2dda2438cd38c609c74d2c33fe/lib/Directives/SubDirective.php#L12-L20

Apparently we even wrote in SubDirective that `figure` is an example of one: https://github.com/doctrine/rst-parser/blob/9107ba8b1dc8dc2dda2438cd38c609c74d2c33fe/lib/Directives/SubDirective.php#L59

Reply:
Thanks for your continuous help 🙏  Sadly I still can't make this work.